Promact.OAuth.Client/src/Promact.OAuth.Client/project.json

```diff
@@ -1,16 +1,23 @@
 {
   "version": "1.0.0-*",
   "dependencies": {
-    "NETStandard.Library": "1.6.0",
+    "Microsoft.AspNetCore.Authentication.OpenIdConnect": "1.1.0",
+    "NETStandard.Library": "1.6.1",
     "Newtonsoft.Json": "9.0.2-beta1",
     "System.Net.Security": "4.0.0"
   },
   "frameworks": {
     "netstandard1.6": {
       "imports": "dnxcore50"
-    },
-    "net461": {
     }
+    //},
```

Comment:
I will removed this commented code. Right now i am getting error while building NET461 in travis, that why I have commented out NET461 framework